docs/docsite/rst/playbook_guide/playbooks_variables.rst

```diff
@@ -112,14 +112,14 @@ While documentation examples focus on ``true/false`` to be compatible with ``ans
 .. table::
    :class: documentation-table
 
-   =============================================================================================== ====================================================================
-    Valid values                                                                                    Description
-   =============================================================================================== ====================================================================
-    ``True`` , ``'true'`` , ``'t'`` , ``'yes'`` , ``'y'`` , ``'on'`` , ``'1'`` , ``1`` , ``1.0``     Truthy values
+   ========================================================================================================== ====================================================================
```

Comment:
Since this table is being updated, it would be good to correct the introductory paragraph to mention that this only applies to `bool` module arguments.

As currently written, this section makes it seem like these rules apply to variables in general, and to parsing of playbooks, inventory, etc. There may even be a better place in the documentation for this table, since it's not about variables, but module arguments.

Reply:
@mattclay getting back to this pr (finally). We don't talk a lot about module arguments in the docs that I can find. We do have https://docs.ansible.com/ansible/latest/module_plugin_guide/modules_intro.html where we could move this section to (Booleans in module arguments could be the new section title).  What do you think?

And if I move it, is there any need to have any replacement text left here for how booleans are handled in variables in general?

Reply:
I think that's an acceptable location to move it to.

There's not really anything to document here about boolean variables, at least not for 2.19+, since Jinja native types are always used. There are boolean behaviors that could be documented elsewhere, but they're not specific to variables.